### PR TITLE
Add link to 'noscript' message

### PR DESCRIPTION
**Pull request by mkljczk**

### CODE REVIEW

**Review by ykzts — APPROVED** [review]

File: config/locales/fa.yml [hunk 1]
```diff
@@ -274,7 +274,6 @@ fa:
       content: تأیید امنیتی انجام نشد. آیا مرورگر شما کوکی‌ها را مسدود می‌کند؟
       title: تأیید امنیتی شکست خورد
     '429': درخواست‌های بیش از حد
-    noscript: برای استفاده از نسخهٔ تحت وب ماستدون، لطفاً جاوااسکریپت را فعال کنید. یا به جایش می‌توانید یک اپ ماستدون را به‌کار ببرید.
```
Comment (Gargron) [inline config/locales/fa.yml]: You have removed many translations which could be kept.

**Review by Gargron — CHANGES_REQUESTED** [review]

File: config/locales/fa.yml (hunk 1 above)
Comment (m4sk1n) [inline config/locales/fa.yml]: but I renamed noscript to noscript_html

Comment (Gargron) [inline config/locales/fa.yml]: These could be renamed to noscript_html

Comment (m4sk1n) [inline config/locales/fa.yml]: @Gargron, now should be fine

File: config/locales/ru.yml [hunk 2]
```diff
@@ -193,7 +193,6 @@ ru:
       content: Проверка безопасности не удалась. Возможно, Вы блокируете cookies?
       title: Проверка безопасности не удалась.
     '429': Слишком много запросов
-    noscript: Для работы с Mastodon, пожалуйста, включите JavaScript.
```
Comment (ykzts) [inline config/locales/ru.yml]: `noscript_html`

File: config/locales/ko.yml [hunk 3]
```diff
@@ -220,7 +220,6 @@ ko:
       content: 보안 인증에 실패했습니다. Cookie를 차단하고 있진 않습니까?
       title: 보안 인증 실패
     '429': 요청 횟수 제한에 도달했습니다.
-    noscript: Mastodon을 사용하기 위해서는 JavaScript를 켜 주십시오.
```
Comment (ykzts) [inline config/locales/ko.yml]: `noscript_html`

File: config/locales/ja.yml [hunk 4]
```diff
@@ -273,7 +273,7 @@ ja:
       content: セキュリティ認証に失敗しました。Cookieをブロックしていませんか？
       title: セキュリティ認証に失敗
     '429': リクエストの制限に達しました。
-    noscript: Mastodonのウェブアプリケーションを利用する場合はJavaScriptを有効にしてください。またはあなたのプラットフォーム向けのMastodonネイティブアプリを探すことができます。
+    noscript_html: Mastodonのウェブアプリケーションを利用する場合はJavaScriptを有効にしてください。またはあなたのプラットフォーム向<a href="https://github.com/tootsuite/documentation/blob/master/Using-Mastodon/Apps.md">けのMastodonネイティブアプリ</a>を探すことができます。
```
Comment (ykzts) [inline config/locales/ja.yml]: broken Japanese translate.

Do not add the A tag except for the locale you know (the translator for each language should do it).

File: config/locales/ca.yml [hunk 5]
```diff
@@ -275,7 +275,6 @@ ca:
       content: La verificació de seguretat ha fallat. Bloquejes les galetes?
       title: La verificació de seguretat ha fallat
     '429': Estrangulat
-    noscript: Per utilitzar Mastodon si us plau activa JavaScript.
```
Comment (ykzts) [inline config/locales/ca.yml]: `noscript_html`

**Review by ykzts — CHANGES_REQUESTED** [review]

File: config/locales/ja.yml (hunk 4 above)
Comment (m4sk1n) [inline config/locales/ja.yml]: Ok

File: config/locales/ca.yml (hunk 5 above)
Comment (m4sk1n) [inline config/locales/ca.yml]: @ykzts now?

**Review by akihikodaki — APPROVED** [review]

File: config/locales/ja.yml [hunk 6]
```diff
@@ -273,8 +273,8 @@ ja:
       content: セキュリティ認証に失敗しました。Cookieをブロックしていませんか？
       title: セキュリティ認証に失敗
     '429': リクエストの制限に達しました。
-    noscript: Mastodonのウェブアプリケーションを利用する場合はJavaScriptを有効にしてください。またはあなたのプラットフォーム向けのMastodonネイティブアプリを探すことができます。
-  exports:
+    noscript_html: Mastodonのウェブアプリケーションを利用する場合はJavaScriptを有効にしてください。またはあなたのプラットフォーム向けのMastodonネイティブアプリを探すことができます。
+    exports:
```
Comment (ykzts) [inline config/locales/ja.yml]: indent?

**Review by ykzts — APPROVED** [review]
has been fixed

**Comment by ykzts** [comment]
@Gargron I think that this was corrected enough. What about you think.

**Review by Gargron — APPROVED** [review]

**Pull request merged**